src/main/java/com/thevoxelbox/voxelsniper/brush/type/performer/OverlayBrush.java

```diff
@@ -156,7 +156,7 @@ private void overlayTwo(Snipe snipe) {
                 int blockY = targetBlock.getY();
                 int blockZ = targetBlock.getZ();
                 for (int y = blockY; y >= editSession.getMinY() && !surfaceFound; y--) { // start scanning from the height you clicked at
-                    if (memory[x + brushSize][z + brushSize] != 1) { // if haven't already found the surface in this column
+                    if (!memory[x + brushSize][z + brushSize]) { // if haven't already found the surface in this column
```

Comment:
The `!` can be overread, maybe a variable helps here :D 
e.x:
```java
var noSurfaceFound = !memory[x + brushSize][z + brushSize];
if(noSurfaceFound) // ...
```

Reply:
FAVS is full of overlapping code horrors :sweat_smile: But yeah, let's improve it step by step, thanks for the review.